agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -77,6 +77,10 @@ backhaul_manager::backhaul_manager(const config_file::sConfigSlave &config,
     configuration_stop_on_failure_attempts = string_utils::stoi(config.stop_on_failure_attempts);
     stop_on_failure_attempts               = configuration_stop_on_failure_attempts;
     LOG(DEBUG) << "stop_on_failure_attempts=" << configuration_stop_on_failure_attempts;
+    m_sConfig.ucc_listener_port = string_utils::stoi(config.ucc_listener_port);
```

Comment:
I think that all certification related members should be moved to a separate class, and the backhaul manager thread should have a member of that class which we can compile out when the time comes and we are product ready.
For the same reason, it makes sense to have the certification parameters in a separate configuration file, not part of the agent configuration file so it won't be accidently moved to the platform DB when we implement #568.

Reply:
I put it there since all the configurations from the configuration file are there. 
We still can compile it out with `#ifdef` where it currently placed.
I don't understand why it can't be moved to DB with all other configurations. Why should it be different?

Reply:
If we can make an optional section in the DB, something like:
```
prplmesh.config.certification
   enable
   port
```
Then fine, I guess it can work.